install_config/certificate_customization.adoc

```diff
@@ -16,8 +16,7 @@ Administrators can configure custom serving certificates for the public host
 names of the {product-title} API and
 xref:../architecture/infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
 This can be done during an
-xref:../install_config/install/advanced_install.adoc#advanced-install-custom-certificates[advanced
-installation] or configured after installation.
+xref:../install_config/install/advanced_install.adoc#advanced-install-custom-certificates[advanced installation] or configured after installation.
```

Comment:
I don't like duplicating content either unless absolutely necessary (and if I did, I'd use "include" statements so the text wouldn't be in there redundantly). I've made this a new section which is closer to what I did for other topics, let me know if you feel this satisfies what you were looking for.